packaging/convert2rhel.spec

```diff
@@ -51,7 +51,7 @@ Requires:       pexpect
 The purpose of the convert2rhel tool is to provide an automated way of
 converting the installed other-than-RHEL OS distribution to Red Hat Enterprise
 Linux (RHEL). The tool replaces all the original OS-signed packages with the
-RHEL ones. Available are conversions of CentOS 6/7/8 and Oracle Linux 6/7/8 to
+RHEL ones. Available are conversions of CentOS Linux 6/7/8 and Oracle Linux 6/7/8 to
```

Comment:
Wrap the line to 80 chars. It's I believe required by the spec file format and we'd be getting warnings when building the rpm.